index.html

```diff
@@ -188,7 +199,7 @@ <h4>Multi-Post <br> Stories</h4>
                     </div>
                 </div>
             </section>
-            <section id="about-container">
+            <section class="about-container">
```

Comment:
Your about section also does not match the Figma template. Consider making it exactly as in the Figma template. The goal is to be as close to the Figma template as it can be.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178109707-9a70a765-833a-479b-a5e2-52da45b26747.png) | ![image](https://user-images.githubusercontent.com/33268927/178109726-b7eb4c09-9fd8-4c76-83bc-1009da5ba9e1.png) |

Reply:
- [ ] @Bantealem great work implementing the changes kindly ensure the layout matches the design, currently the languages side is bigger kindly reduce its width:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/178119670-45f62943-8d13-4263-bc63-b30e64fe2ade.png) |  ![image](https://user-images.githubusercontent.com/61408860/178119677-e19e6b25-e665-4de3-aaa4-8e729605b855.png)